src/Core/Services/Implementations/InMemoryServiceBusApplicationCacheService.cs

```diff
@@ -19,38 +20,38 @@ public class InMemoryServiceBusApplicationCacheService : InMemoryApplicationCach
         : base(organizationRepository, providerRepository)
     {
         _subName = CoreHelpers.GetApplicationCacheServiceBusSubscriptionName(globalSettings);
-        _topicClient = new TopicClient(globalSettings.ServiceBus.ConnectionString,
-            globalSettings.ServiceBus.ApplicationCacheTopicName);
+        _serviceBusClient = new ServiceBusClient(globalSettings.ServiceBus.ConnectionString);
+        _topicMessageSender = new ServiceBusClient(globalSettings.ServiceBus.ConnectionString).CreateSender(globalSettings.ServiceBus.ApplicationCacheTopicName);
```

Comment:
❓ Is it safe to call `CreateSender` in the constructor?

Reply:
Yes, that's the recommended practice: https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-performance-improvements?tabs=net-standard-sdk-2#reusing-factories-and-clients